frontend/tests/_bootstrap.php

```diff
@@ -7,3 +7,7 @@
 require_once YII_APP_BASE_PATH . '/vendor/yiisoft/yii2/Yii.php';
 require_once YII_APP_BASE_PATH . '/common/config/bootstrap.php';
 require_once __DIR__ . '/../config/bootstrap.php';
+
+//codeception namespace
+\Yii::setAlias('@frontend/tests/Helper', '@frontend/tests/_support/Helper');
```

Comment:
This one looks a bit wrong as well. I'd name it `support` and map whole dir in case more helpers will get in.

Reply:
This is codeception template. 
I can see `Helper` (not `Helpers`) and `Page` inside this folder.

I can't remember reason for avoid `_support` level from namespaces

Reply:
@samdark I try it now. 

This is `Codeception` magic: command to generate helper produced output like this
```sh
root@b46a5e9d9643:/var/www/html/frontend# codecept generate:helper TestHelper
Helper /var/www/html/frontend/tests/_support/Helper/TestHelper.php created
root@b46a5e9d9643:/var/www/html/frontend# cat /var/www/html/frontend/tests/_support/Helper/TestHelper.php
<?php
namespace frontend\tests\Helper;

// here you can define custom actions
// all public methods declared in helper class will be available in $I

class TestHelper extends \Codeception\Module
{

}
```

Reply:
same situation with `step`
[see StepObjects in this page](http://codeception.com/docs/06-ReusingTestCode)
```sh
StepObject was created in /tests/acceptance/_support/Step/Acceptance/Admin.php
```
```php
namespace Step\Acceptance;
```

Reply:
I see... OK.